deploy/openshift/releases/R2011/manifests/00-contrail-09-manager.yaml

```diff
@@ -121,7 +121,7 @@ spec:
           - name: init
             image: python:3.8.2-alpine
           - name: provisioner
-            image: hub.juniper.net/contrail/contrail-operator-provisioner:2011.31
+            image: hub.juniper.net/contrail/contrail-operator-provisioner:2011.31-ubi
```

Comment:
Could you change `hub.juniper.net/contrail` to `hub.juniper.net/contrail-nightly` in the release manifests as part of this PR? I think it would be better this way, as then we can actually tests those release manifests.

Reply:
Oki doki!